flow/util/config.py

```diff
@@ -15,6 +15,7 @@
 status_performance_warn_threshold = float(default=0.2)
 show_traceback = boolean()
 eligible_jobs_max_lines = int(default=10)
+parallel_nature = str(default='thread')
```

Comment:
I do not think that this is a good config variable name, because it is too general and the wording is somewhat informal. I propose: *status_parallelization*.

Reply:
Will change the name